File: Hyeona/없는_숫자_더하기.java
package Hyeona;

public class 없는_숫자_더하기 {
    public int solution(int[] numbers) {
        int answer = 45;
        int []arr=new int[] {0,1,2,3,4,5,6,7,8,9};
        for(int i=0;i<numbers.length;i++)
        {
            for(int j=0;j<arr.length;j++)
            {
                if(numbers[i]==arr[j])
                    answer=answer-j;
            }
        }
        return answer;
    }
}

File: yeonkyu/없는_숫자_더하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// numbers_len은 배열 numbers의 길이입니다.
int solution(int numbers[], size_t numbers_len) {
    int answer = -1;
    int sum = 45;
    for (int i = 0; numbers_len > i; i++) {
        sum -= numbers[i];
    }
    answer = sum;
    return answer;
}
